packages/flutter_tools/lib/src/test/test_time_recorder.dart

```diff
@@ -0,0 +1,81 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import '../globals.dart' as globals;
```

Comment:
Please don't add additional classes that depend on globals. Provide a logger in the constructor of `TestTimeRecorder`

Reply:
Done.